cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -37,12 +37,12 @@
 import java.util.jar.JarEntry;
 import java.util.jar.JarInputStream;
 import java.util.stream.Stream;
+import java.util.zip.ZipEntry;
```

Comment:
There is also a ZipEntry class provided by  `org.apache.commons.compress.archivers.zip.ZipFile;`. Is there any difference?